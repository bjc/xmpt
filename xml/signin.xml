<stream:stream xmlns:stream='http://etherx.jabber.org/streams'
	       xmlns:xml='http://www.w3.org/XML/1998/namespace'
	       xmlns='jabber:client'
	       xml:lang='en'
	       version='1.0'
	       to='{domainpart}'>
