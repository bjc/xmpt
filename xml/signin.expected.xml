<stream:stream xmlns:stream='http://etherx.jabber.org/streams'
	       version='1.0'
	       from='{domainpart}'
	       id='.*'
	       xmlns='jabber:client'>
  <stream:features>
    <mechanisms xmlns='urn:ietf:params:xml:ns:xmpp-sasl'>
      <mechanism>SCRAM-SHA-1</mechanism>
      <mechanism>PLAIN</mechanism>
    </mechanisms>
    <auth xmlns='http://jabber.org/features/iq-auth'/>
  </stream:features>
